### DISCUSSION TITLE
Storing result of user in db

### DISCUSSION THREAD

**Discussion by mouad-talhaoui** [Lumieducation/H5P-Nodejs-library#2308] [Q&A]

**Comment by sr258** [comment]
I'm working on this feature here: https://github.com/Lumieducation/H5P-Nodejs-library/pull/1886

**Comment by sr258** [comment]
The feature is now part of 9.2.0. See the docs here: https://docs.lumi.education/advanced-usage/user-content-state https://docs.lumi.education/advanced-usage/completion-tracking
